airbyte-integrations/connectors/source-plaid/source_plaid/manifest.yaml

```diff
@@ -240,23 +265,28 @@ streams:
           pending:
             type:
               - boolean
-              - "null"
           pending_transaction_id:
             type:
               - string
               - "null"
           personal_finance_category:
-            type:
-              - string
-              - "null"
+            properties:
+              primary:
+                type:
+                  - string
+              detailed:
+                type:
+                  - string
+              confidence_level:
+                type:
+                  - string
+                  - "null"
```

Comment:
nit: do we want to add 
```
type:
- object
- "null"
```
 at the `properties level`? One indent in from `personal_finance_category`

Reply:
We sure do! Thanks for spotting that.